File: packages/sdk-java/src/main/java/com/zenvio/sdk/model/SmsSendResponse.java
package com.zenvio.sdk.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class SmsSendResponse {
    private boolean success;
    private Data data;

    public boolean isSuccess() { return success; }
    public void setSuccess(boolean success) { this.success = success; }
    public Data getData() { return data; }
    public void setData(Data data) { this.data = data; }

    @JsonIgnoreProperties(ignoreUnknown = true)
    public static class Data {
        private String status;
        private int count;
        private java.util.List<String> smsIds;
        private String scheduledAt;

        public String getStatus() { return status; }
        public void setStatus(String status) { this.status = status; }
        public int getCount() { return count; }
        public void setCount(int count) { this.count = count; }

        @com.fasterxml.jackson.annotation.JsonProperty("sms_ids")
        public java.util.List<String> getSmsIds() { return smsIds; }
        @com.fasterxml.jackson.annotation.JsonProperty("sms_ids")
        public void setSmsIds(java.util.List<String> smsIds) { this.smsIds = smsIds; }

        @com.fasterxml.jackson.annotation.JsonProperty("scheduled_at")
        public String getScheduledAt() { return scheduledAt; }
        @com.fasterxml.jackson.annotation.JsonProperty("scheduled_at")
        public void setScheduledAt(String scheduledAt) { this.scheduledAt = scheduledAt; }
    }
}

File: packages/sdk-dotnet/src/Zenvio/Models/Sms/SmsSendResponse.cs
using System.Text.Json.Serialization;

namespace Zenvio.Models.Sms;

public record SmsSendResponse(
    [property: JsonPropertyName("success")] bool Success,
    [property: JsonPropertyName("data")] SmsSendData Data
);

public record SmsSendData(
    [property: JsonPropertyName("status")] string Status,
    [property: JsonPropertyName("count")] int? Count,
    [property: JsonPropertyName("sms_ids")] List<string> SmsIds,
    [property: JsonPropertyName("scheduled_at")] string? ScheduledAt = null
);
